ocean_lib/models/bpool.py

```diff
@@ -403,14 +448,15 @@ def join_swap_extern_amount_in(
             from_wallet,
         )
 
+    @enforce_types
     def exit_swap_pool_amount_in(
         self,
         pool_amount_in: int,
         min_amount_out: int,
         from_wallet: Wallet,
     ) -> str:
         """
-        Pay `poolAmountIn` pool shares into the pool, getting `tokenAmountOut`
+        Pay `poolAmountIn` pool shares into the poopool_template_addresspool_template_addressl, getting `tokenAmountOut`
```

Comment:
I think there's a typo here, it's `poopool`

Reply:
Fixed.